Lib/test/test_dictviews.py

```diff
@@ -214,6 +214,16 @@ def test_items_set_operations(self):
         self.assertTrue(de.items().isdisjoint(de.items()))
         self.assertTrue(de.items().isdisjoint([1]))
 
+    def test_set_operations_with_iterator(self):
+        origin = {1: 2, 3: 4}
+        self.assertEqual((origin.keys() | iter([1, 2])), {1, 2, 3})
```

Comment:
Why no `&` for keys?

Reply:
Parenthesis around the expression are redundant.

Reply:
> Why no & for keys?

Oops, That's my mistake.

> Parenthesis around the expression are redundant.

I've updated it :)